.rubocop_todo.yml

```diff
@@ -1,137 +1,476 @@
-# This configuration was generated by `rubocop --auto-gen-config`
-# on 2014-07-21 12:48:09 +0200 using RuboCop version 0.24.1.
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-02-08 14:42:45 +0100 using RuboCop version 0.37.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
+# Offense count: 2
+Lint/DuplicatedKey:
```

Comment:
Yeap, is it for sth like this line?
https://github.com/openSUSE/osem/blob/master/db/migrate/20140801164901_move_conference_media_to_commercial.rb#L13